internal/cmd/stackset/stackset.go

```diff
@@ -0,0 +1,60 @@
+package stackset
+
+import (
+	"github.com/aws-cloudformation/rain/internal/config"
+	"github.com/aws-cloudformation/rain/internal/console"
+	"github.com/spf13/cobra"
+)
+
+const usageTemplate = `Usage:{{if .Runnable}}
+  <cyan>{{.UseLine}}</>{{end}}{{if .HasAvailableSubCommands}}
+  <cyan>{{.CommandPath}}</> [<gray>command</>]{{end}}{{if gt (len .Aliases) 0}}
+
+Aliases: {{.NameAndAliases}}{{end}}{{if .HasExample}}
+
+Examples:
+{{.Example}}{{end}}{{if .HasAvailableSubCommands}}
+
+Available Commands:{{range .Commands}}{{if (or .IsAvailableCommand (eq .Name "help"))}}
+  <cyan>{{rpad .Name .NamePadding }}</> {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableLocalFlags}}
+
+Flags:
+{{.LocalFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasAvailableInheritedFlags}}
+
+Global Flags:
+{{.InheritedFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasHelpSubCommands}}
+
+Additional help topics:{{range .Commands}}{{if .IsAdditionalHelpTopicCommand}}
+{{rpad .CommandPath .CommandPathPadding}} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableSubCommands}}
+
+Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
+`
+
+func addCommand(profileOptions bool, c *cobra.Command) {
+	if profileOptions {
+		c.Flags().StringVarP(&config.Profile, "profile", "p", "", "AWS profile name; read from the AWS CLI configuration file")
+		c.Flags().StringVarP(&config.Region, "region", "r", "", "AWS region to use")
+	}
+
+	StackSetCmd.AddCommand(c)
+}
+
+var StackSetCmd = &cobra.Command{
+	Use:   "stackset <stack_set command>",
+	Short: "This command allows to manipulate stack sets.",
+	Long:  "This command allows to manipulate stack sets. It has no action if specific stack set command is not added.",
+}
+
+func init() {
+	addCommand(true, StackSetLsCmd)
+	addCommand(true, StackSetDeployCmd)
+	addCommand(true, StackSetRmCmd)
+
+	oldUsageFunc := StackSetCmd.UsageFunc()
```

Comment:
I'm a little confused about what's going on here, can you add a comment?

Reply:
Here we add sub commands to a stackset command